client/package.json

```diff
@@ -5,6 +5,7 @@
     "normalize.css": "5.0.0",
     "preact": "^7.1.0",
     "preact-compat": "^3.9.2",
+    "react-markdown": "^2.4.4",
```

Comment:
Ooops, actualy, it's used here : https://github.com/jacquarg/konnectors/blob/61d38892dd43284bca15ecef349fb281d44c07d7/client/app/components/AccountConnection.jsx#L18-L23